packages/client/test/net/peer/rlpxpeer.spec.ts

```diff
@@ -89,9 +89,9 @@ tape('[RlpxPeer]', async (t) => {
     peer.config.events.on(Event.PEER_DISCONNECTED, (rlpxPeer) =>
       t.equals(rlpxPeer.pooled, false, 'got disconnected')
     )
-    peer.rlpx!.emit('peer:error', rlpxPeer, new Error('err0'))
-    peer.rlpx!.emit('peer:added', rlpxPeer)
-    peer.rlpx!.emit('peer:removed', rlpxPeer, 'reason')
+    peer.rlpx!.events.emit('peer:error', rlpxPeer, new Error('err0'))
```

Comment:
I think the tests in this file are failing because the mock `devp2p` peer the test is connecting to doesn't have the `events` property on it.  So, you'd need to adjust the properties of the `RLPx` class that starts on line 11 to resolve I think.  That said, I say we correct this once we merge the `vitest` PR so we don't have to spend lots of time rewriting these tests later.